Exercise 19-JS/index.html

```diff
@@ -0,0 +1,12 @@
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Exercise 19</title>
+    <script src="index.js">
+        alert()
```

Comment:
Here you add reference to external file index.js
And also there is an empty alert() inside.
hmmm... looks a bit strange 😅